README.md

```diff
@@ -11,7 +11,9 @@ In `mix.exs`, add the ExMachina dependency:
 
 ```elixir
 def deps do
-  [{:ex_machina, "~> 0.4"}]
+  [
+    {:ex_machina, "~> 0.4", only: "test"}
```

Comment:
I like commas in the README so you can add it anywhere in your project without modification, but I think it might be a WTF factor to those who haven't seen the light ;)